apps/teams/tests/test_creation.py

```diff
@@ -24,12 +25,28 @@ def test_create_for_user(self):
         assert [SUPER_ADMIN_GROUP] == [group.name for group in membership.groups.all()]
 
 
-def test_create_team_view(db, client):
+@pytest.mark.django_db()
+def test_slug_generation_on_team_creation(db, client):
```

Comment:
Since we're now using `@pytest.mark.django_db()`, you can remove the `db` parameter